.github/actions/detect-workflow/Dockerfile

```diff
@@ -12,6 +12,11 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+# NOTE: Updates to the golang base image sha256 should be verified by checking
+# the Docker official images tag details.
+# https://raw.githubusercontent.com/docker-library/repo-info/master/repos/golang/tag-details.md
+
+# tag=1.18.4
```

Comment:
is this for Renovate? Should the digest below (line 34) have a similar annotation?

Reply:
I'm not sure this will actually work yet, I asked about it here:
https://github.com/slsa-framework/slsa-github-generator/pull/557#issuecomment-1189937390

I can remove it for now and add it back when I get confirmation that it should work. I can't find any docs that talk about these tag comments.

As for the one below, disroless doesn't have tags as far as I can tell so we need to stick with just the sha. That's generally ok I think as long as we can verify it's signature.

Reply:
AFAIK, docker support `FROM name:v1.2.3@sha256:xxx`, does it not?

Reply:
Yeah, that might be the right way. I don't think Docker validates it but it would be good if renovate bot updated it.

Reply:
I updated this to <name>:<tag>@<sha>